### ISSUE TITLE
Footer widget headings should be an h2 instead of an h3

### ISSUE THREAD

**Issue by andrewmarino** [unitymakesus/justgetsimple#53]
These are being registered in the theme.

**Comment by andrewmarino** [comment]
This one is a little tricky... we could potentially put a visually hidden h2 in the global footer (without it affecting existing headings in all the sites—some sites use styles for h3).

Or we could bite the bullet and change the h3s to h2s and fix any styles on each site.
